imports/server/gdriveChangeFetcher.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable camelcase */
+import { Meteor } from 'meteor/meteor';
+import type { drive_v3 } from '@googleapis/drive';
+import Flags from '../Flags';
+import DocumentActivities from '../lib/models/DocumentActivities';
+import Documents from '../lib/models/Documents';
+import FeatureFlags from '../lib/models/FeatureFlags';
+import DriveClient from './gdriveClientRefresher';
+import ignoringDuplicateKeyErrors from './ignoringDuplicateKeyErrors';
+import DriveChangesPageTokens from './models/DriveChangesPageTokens';
+import Locks, { PREEMPT_TIMEOUT } from './models/Locks';
+
+const ACTIVITY_GRANULARITY = 5 * 60 * 1000; // milliseconds
+
+async function recordDriveChange(file: Pick<drive_v3.Schema$File, 'id' | 'modifiedTime'>) {
+  if (!file.id || !file.modifiedTime) {
+    return;
+  }
+
+  const document = Documents.findOne(file.id);
```

Comment:
I don't think Google's notion of the `file.id` is going to match our `Document`'s  `_id`.  I would expect it to match [`value.id`](https://github.com/deathandmayhem/jolly-roger/blob/18609e4a627f15766e6e24f4b7b365c9462fd887/imports/lib/schemas/Document.ts#L22).

Reply:
D'oh. Fixed.

Reply:
(And confirmed in local testing that the end-to-end does actually capture `DocumentActivity` records)